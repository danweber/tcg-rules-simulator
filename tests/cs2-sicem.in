VERSION:2
# [Main] One of your Monsters gains +4000 DP for the turn. Then, one of your Monsters attacks.
FIELD2:Mon1,CS2-02 Mon2,CS2-02
FIELD1:Other,CS1-02,REST
HAND2:CS2-Sicem
SECURITY1:CS1-02
DECK1:CS1-02
GAME:P2 T3 MAIN 9
DUMP:DP,SECURITY,HAND


ECHO: -- use card. 
CMD:PLAY CS2-Sicem
EFFECT-TREE
GET-CHOICES
DUMP:DP

ECHO: -- Give bonus to first...
STEP:TO:0
EFFECT-TREE
GET-CHOICES
DUMP:DP,SECURITY


ECHO: -- ... and attack with second, into player.
STEP:TO:3
EFFECT-TREE
GET-CHOICES
DUMP:DP,SECURITY

EXIT
