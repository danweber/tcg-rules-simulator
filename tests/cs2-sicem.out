FIELD1:Other,CS1-Papershield,REST,500DP
FIELD2:Mon1,CS2-Sharpblade,2000DP Mon2,CS2-Sharpblade,2000DP
SECURITY1:CS1-Papershield
SECURITY2:
HAND1:
HAND2:CS2-Sicem
GAME:P2 T3 MAIN 9
ECHO: -- use card.
EFFECT-TREE:DSE DEPTH 1 DONE:use Sic 'Em!
EFFECT-TREE:RES DEPTH 2 PROCESSING:Sic 'Em!
GET-CHOICES:Choose 1 target for getting DP:
GET-CHOICES:Target Sharp Blade 1
GET-CHOICES:Target Sharp Blade 2
FIELD1:Other,CS1-Papershield,REST,500DP
FIELD2:Mon1,CS2-Sharpblade,2000DP Mon2,CS2-Sharpblade,2000DP
GAME:P2 T3 MAIN 3
ECHO: -- Give bonus to first...
EFFECT-TREE:DSE DEPTH 1 DONE:use Sic 'Em!
EFFECT-TREE:RES DEPTH 2 PROCESSING:Sic 'Em!
GET-CHOICES:Choose attack:
GET-CHOICES:Attack Sharp Blade into Paper Shield
GET-CHOICES:Attack Sharp Blade into player
GET-CHOICES:Attack Sharp Blade into Paper Shield
GET-CHOICES:Attack Sharp Blade into player
FIELD1:Other,CS1-Papershield,REST,500DP
FIELD2:Mon1,CS2-Sharpblade,6000DP Mon2,CS2-Sharpblade,2000DP
SECURITY1:CS1-Papershield
SECURITY2:
GAME:P2 T3 MAIN 3
ECHO: -- ... and attack with second, into player.
EFFECT-TREE:NOTHING
GET-CHOICES:Choose attack:
FIELD1:Other,CS1-Papershield,REST,500DP
FIELD2:Mon1,CS2-Sharpblade,6000DP Mon2,CS2-Sharpblade,REST,2000DP
SECURITY1:
SECURITY2:
GAME:P2 T3 MAIN 3
